snuba/clusters/cluster.py

```diff
@@ -47,7 +47,13 @@ class ClickhouseClientSettings(Enum):
     CLEANUP = ClickhouseClientSettingsType({}, None)
     INSERT = ClickhouseClientSettingsType({}, None)
     MIGRATE = ClickhouseClientSettingsType(
-        {"load_balancing": "in_order", "replication_alter_partitions_sync": 2}, 10000
+        {
+            "load_balancing": "in_order",
+            "replication_alter_partitions_sync": 2,
+            "mutations_sync": 2,
+            "database_atomic_wait_for_drop_and_detach_synchronously": 1,
```

Comment:
Do we know that these settings work for  clickhouse version `20.7` ? If not, we maybe have to have a different setting enum

Reply:
In my testing everything seemed fine on 20.7